packages/components/pie-divider/src/index.ts

```diff
@@ -29,8 +32,8 @@ export class PieDivider extends LitElement implements DividerProps {
                 data-test-id="pie-divider"
                 aria-hidden="true"
                 class="c-divider"
-                variant=${variant || 'default'}
-                orientation=${orientation || 'horizontal'}
+                variant=${ifDefined(variant || defaultProps.variant)}
```

Comment:
Out of interest, was there a warning in your editor that meant you needed to add `ifDefined` here?

Reply:
Nope, truly excessive indeed 🤣